review (#6)
ScreaMy7/owasp-mastg: demos/android/MASVS-CODE/MASTG-DEMO-0042/MASTG-DEMO-0042.md

---
platform: android
title: Implicit Intent Hijacking
id: MASTG-DEMO-0042
code: [kotlin]
test: MASTG-TEST-0026
---

### Sample

> The code snippet shows that an implicit intent is being used to launch an activity using Intent without specifying the target component. This may allow malicious applications to hijack the intent.

Reviewer: ```suggestion
The code snippet shows that an implicit intent is used to launch an activity using Intent without specifying the target component. This could allow malicious applications to hijack the intent.
```
